gradle.properties

```diff
@@ -21,4 +21,4 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
-agent_version=0.9.0
\ No newline at end of file
+agent_version=0.7.0
```

Comment:
This should help with testing the bump automation in https://github.com/elastic/opbeans-android/pull/5. In addition, `0.9.0` is not available yet.